_projects/02-foomatic.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Foomatic"
+---
+Foomatic is a database-driven system for integrating free software printer drivers with common spoolers under Unix. It supports CUPS, LPRng, LPD, GNUlpr, Solaris LP, PPR, PDQ, CPS, and direct printing with every free software printer driver known to us and every printer known to work with these drivers. 
```

Comment:
Note that due to the fact that all printing systems/spoolers except CUPS are not maintained any more we also have stopped supporting them with Foomatic. The universal `foomatic-rip` filter of the foomatic-filters package got replaced by a simplified one which supports only CUPS and spoolerless, direct printing (for development/debugging).
Especially this means that the introduction has to be changed telling that we only support CUPS and driverless printing and foomatic-filters has to get removed from the list of the packages.

Reply:
I have updated the content.